### DISCUSSION TITLE
Running Average of my accuracy, losses etc.

### DISCUSSION THREAD

**Discussion by karanchahal** [Lightning-AI/pytorch-lightning#5819] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
#### What is your question?    
I want my tqdm logger to show me a history of my training on the terminal. Right now, when a epoch ends, all data for it is scrubbed from the command line and the new epoch data is shown.

Also I want to see the running accuracy of my network and a running average of my loss on the tqdm bar. How should I go on about doing that ?
#### What have you tried?    
I have looked at the docs and logging but am unable to figure out how to modify the tqdm logger, more so maintain a running average
#### What's your environment?   

- conda version: latest
- PyTorch version   : 1.3
- Lightning version : pip install pytorch-lightning at the date of this issue  
- Test-tube version: came boot strapped with lightning.

I installed everything on the date of this issue.

**Comment by s-rog** [comment, marked as answer]
Use tensorboard!

For running averages you have to implement the logic in training step

**Comment by karanchahal** [comment]
Thank you, yes I've figured it out now !
